vagga.yaml

```diff
@@ -85,3 +97,7 @@ commands:
     children:
       app: *run-app
       db: *run-db
+  test: !Command
+    description: Run tests
+    container: test
+    run: pre-commit run --all-files
```

Comment:
please expose separate `vagga pre-commit` command using `exec` syntax (`run` value is a sequeance)